app/src/main/java/org/thunderdog/challegram/telegram/TdlibCache.java

```diff
@@ -1898,6 +1913,7 @@ private int putSupergroup (TdApi.Supergroup supergroup) {
 
   private boolean putSupergroupFull (long supergroupId, TdApi.SupergroupFullInfo supergroupFull) {
     supergroupsFulls.put(supergroupId, supergroupFull);
+    supergroupsFullsLastUpdateTime.append(supergroupId, SystemClock.uptimeMillis());
```

Comment:
Incorrect method used.

```suggestion
    supergroupsFullsLastUpdateTime.put(supergroupId, SystemClock.uptimeMillis());
```